pkg/microservice/aslan/core/common/repository/models/sonar.go

```diff
@@ -0,0 +1,13 @@
+package models
+
+import "go.mongodb.org/mongo-driver/bson/primitive"
+
+type SonarIntegration struct {
+	ID            primitive.ObjectID `bson:"_id,omitempty"`
+	ServerAddress string             `bson:"server_address"`
+	Token         string             `bson:"token"`
+}
+
+func (SonarIntegration) TableName() string {
+	return "sonar_integration"
```

Comment:
Could we reuse `system_setting` table?

Reply:
I think it is not a setting. It is a system integration information, and it deserves a single table.